paket-files/haf/YoLo/YoLo.fs

```diff
@@ -622,7 +622,7 @@ module App =
 
   /// Get the assembly resource
   let resource name =
-#if DNXCORE50
+#if NETSTANDARD1_5
```

Comment:
@haf @theimowski about `DNXCORE50` `compilationOptions.define`, you can add that meanwhile (for yolo.fs), so the directive is passed to compiler

```json
{
 "compilationOptions": {
     "define": [ 
          "DNXCORE50",
     ]
}
```